### ISSUE TITLE
Hooks are not executed in zsh

### ISSUE THREAD

**Issue by hinricht** [sbstp/kubie#170]
Given the following config: 

```
hooks:
  start_ctx: >
    notify-send "Kubie activated context $(kubie info ctx)"
  stop_ctx: >
    notify-send "Kubie deactivated context $(kubie info ctx)"
```

* In bash, notifications are shown properly
* In zsh, I don't see any notifications (although I can manually reate a notification with the same cmd)

I also tried with other cmds, none of them worked in zsh.

**Comment by idebeijer** [comment]
Should be fixed in #190, the code to run the hooks for zsh wasn't there yet

**Comment by hinricht** [comment]
@idebeijer Awesome, thanks! Do you mind releasing a new version including this fix ?

**Comment by hinricht** [comment]
@sbstp Sorry I tagged the wrong person:

> Awesome, thanks! Do you mind releasing a new version including this fix ?

**Comment by nscherer-chipotle** [comment]
@sbstp Hello! Unless I'm mistaken, this didn't make it into v0.23.1 despite the merged PR. (I don't see the file changed in #190 in the commit diff). Could a new release be made with this fix when you get a chance, please?

**Comment by sbstp** [comment]
Not sure I get what you mean @nscherer-chipotle . PR #190 was merged to master properly, and it's included in the releases since v0.23.0. Are you saying the PR was incomplete?

**Comment by nscherer-chipotle** [comment]
My bad! For some reason I didn't think this made it into v.0.23.0. Now that I look closer it clearly did. Evidently I just have something wrong locally that's preventing my zsh hooks from working. Apologies for the confusion/wasting your time!
